lib/bindings/http/binary_0_1.js

```diff
@@ -1,7 +1,7 @@
 var axios = require("axios");
 
 function HTTPBinary(configuration){
-  this.config = configuration;
+  this.config = JSON.parse(JSON.stringify(configuration));
```

Comment:
Please check out the [line #14](https://github.com/cloudevents/sdk-javascript/blob/77bad8d07f3c707446a5414d0a1e658c2752f8cf/lib/bindings/http/binary_0_1.js#L14). I suggest you remove this copy because it is useless.

Reply:
I saw line #14. I don't think it's useless because the config object is being changed in line 6/7. That's what is causing npm test to fail...

Reply:
It might make more sense if you look at my other pull request first. https://github.com/cloudevents/sdk-javascript/pull/15